project/profiles/capacity4more/behat/features/group_content_access.feature

```diff
@@ -35,3 +35,19 @@ Feature: Group content access
     When  I am logged in as user "isaacnewton"
     Then  I visit "Test content" node of type "discussion"
     And   I should see "Access denied"
+
+  @javascript
+  Scenario: An anonymous user shouldn't be able to see a restricted group's content.
+    Given I am logged in as user "turing"
+    And   I change access of group "My test group" to "Restricted" with the domain "turingmachine.com"
+    When  I am an anonymous user
+    Then  I visit "Test content" node of type "discussion"
+    And   I should not see "Test content"
```

Comment:
Anon users are redirected to the login page. I will add another test to check for `Access Denied` If a user **is** logged in but not authorized to view the content.